src/Rfc6455/Handshake/HandshakeInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This file is a part of Woketo package.
+ *
+ * (c) Nekland <dev@nekland.fr>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+
+namespace Nekland\Woketo\Rfc6455\Handshake;
+
+use Nekland\Woketo\Exception\WebsocketException;
+use Nekland\Woketo\Http\AbstractHttpMessage;
+use Nekland\Woketo\Http\Request;
+use Nekland\Woketo\Http\Response;
+
+interface HandshakeInterface
+{
+    const GUID = '258EAFA5-E914-47DA-95CA-C5AB0DC85B11';

```

Comment:
It's not related to the WebSocket RFC ! But I'll add this to the handshake methods.